app/services/rm_api_service.rb

```diff
@@ -18,6 +18,18 @@ def request(verb, path, body = nil)
     end
   end
 
+  def request_multi(pending)
```

Comment:
There is no `funcadelic.rb`, but this is almost certainly a `Monoid`. Should there be? 🎸

Reply:
https://images-na.ssl-images-amazon.com/images/I/91H5uzqlvgL._SL1500_.jpg

_if you hear any noise, it's just me and monoids_